cdap-ui/graphql/schema/applicationSchema.graphql

```diff
@@ -0,0 +1,61 @@
+"""
+ Copyright © 2019 Cask Data, Inc.
+ 
+ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ use this file except in compliance with the License. You may obtain a copy of
+ the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ License for the specific language governing permissions and limitations under
+ the License.
+"""
+
+# import ArtifactSummary from "artifactSchema.graphql"
+# import ProgramRecord from "programRecordSchema.graphql"
+# import Metadata from "metadataSchema.graphql"
+
+type ApplicationRecord {
+
```

Comment:
nit: Is there any specific reason why we have newline between all the fields across all the schemas? It was a little hard for me to read it as one schema object.

Reply:
That's the format used in .graphql files. We can definitely change them

Reply:
Yes. Please change them. Lets not have the extra new line between each field of the schema. @elfenheart Let me know if you think otherwise.